trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+# Defines a trimmer_decorator class
+class TrimmerDecorator < BaseDecorator
```

Comment:
Also here, please require the `BaseDecorator` class from the `base_decorator.rb` file using `require_relative` to avoid the **NameError**.

Reply:
Implemented